lib/plugins/receipts.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
-module.exports = function (client) {
+module.exports = function (client, stanzas, config) {
+
+    var sendReceipts =  config.sendReceipts || true;
```

Comment:
This will always be `true`.

I think `sendReceipts = config.sendReceipts !== false` should work.